crates/driver/src/tests/setup/solver.rs

```diff
@@ -272,7 +279,104 @@ struct StateInner {
     /// Has this solver been called yet? If so, attempting to make another call
     /// will result in a failed test.
     called: bool,
+    /// Forwards the notifications received by the solver.
+    notifications: std::sync::mpsc::Sender<serde_json::Value>,
 }
 
 #[derive(Debug, Clone)]
 struct State(Arc<Mutex<StateInner>>);
+
+#[allow(dead_code)]
+pub mod dto {
+    //! Data transfer objects for the solver engine API.
```

Comment:
Not sure this will work out.
Do you think it makes sense to have a reminder CI job that checks for these kinds of things like you added for the DB readme?

Reply:
If we add a new notification to driver but forget to add to this test enum, we still have the same issue right? if so, I would prefer to have the reminder as Martin suggested.